src/ddmd/statement.d

```diff
@@ -172,19 +190,33 @@ extern (C++) abstract class Statement : RootObject
         return this;
     }
 
+    /****************************
+     * Determine if an enclosed `break` would apply to this
+     * statement, such as if it is a loop or switch statement.
+     * Returns:
+     *  `true` if it does
```

Comment:
The indentation of the `Returns:` sections are not consistent. This one is indented one space.